test/e2e/source_test.go

```diff
@@ -0,0 +1,92 @@
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"knative.dev/eventing-rabbitmq/test/e2e/config/recorder"
+	"knative.dev/eventing-rabbitmq/test/e2e/config/source"
+	"knative.dev/eventing-rabbitmq/test/e2e/config/sourceproducer"
```

Comment:
Aren't you using the eventshub (fka recorder) from reconciler-test here? https://github.com/knative-sandbox/reconciler-test/blob/master/test/example/recorder_feature.go